### ISSUE TITLE
Solo mining in 2021 - Lottery ticket style

### ISSUE THREAD

**Issue by talvasconcelos** [luke-jr/bfgminer#825]
So, I want to fire up my Antminers for lottery ticket style solo mining, on a RPI!

Thing is, I don't want to have another full node for this. I have a full, unpruned node (latest 0.22.0) already running on the LAN, I wanted to use that to connect to and just run BFGminer (I'm actually trying cgminer now) on a RPI on the same network.

What are my options? @luke-jr ?

**Comment by luke-jr** [comment]
Should "just work"

**Comment by talvasconcelos** [comment]
`--generate-to bc1xxxxxxxxxx` spits out invalid address!! if i don't set a generate-to address and I get REALLY lucky and find a block, where would the reward go? the remote/lan bitcoin core i'm using for mining?

**Comment by luke-jr** [comment]
Yes, Bech32 isn't supported yet. Just use a legacy/Base58 address.

**Comment by talvasconcelos** [comment]
I suspected that, but something else is wrong. Bfgminer doesn't seem to
connect to my node on the lan (cgminer does but doesn't do GBT), and also,
even if I try to connect to a pool just to see if things work, the pi can't
find the miners.

**Comment by talvasconcelos** [comment]
Sorry for keep coming back to this... I have a couple, just want to use 1 actually, antminers U1 and U2
Just installed raspberry pi os, downloaded bfgminer 5.5.0, i can connect to my LAN node, etc... only problem is that th RPI doesn't even recognize the USB miners, bfgminer doesn't find them, so no hashes.

lsusb doesn't get them `sudo ls /dev/*USB*` returns nothing! the chip on these things is BM 1380 U020114E2, mayve the pi doesn't have the drivers! I'll try tomorrow on a windows machine, just to rule out the chance that the miners are dead... they do get hot as hell though! 

Can I get any help on how to have raspian recognize the miners?

**Comment by luke-jr** [comment]
You probably need a USB driver installed. Try `ftdi_sio`, `cp210x`, `pl2303`

**Comment by talvasconcelos** [comment]
After modprobing those and running bfgminer with -D -T --verbose outputs this:
```
[2021-10-28 10:16:28] setrlimit: Soft fd limit not being changed from 1024 (FD_SETSIZE=1024; hard limit=1048576)
 [2021-10-28 10:16:28] Started bfgminer 5.5.0
 [2021-10-28 10:16:28] hidapi_try_lib: Couldn't load libhidapi.so: libhidapi.so: cannot open shared object file: No such file or directory
 [2021-10-28 10:16:28] hidapi_try_lib: Couldn't load libhidapi-0.so: libhidapi-0.so: cannot open shared object file: No such file or directory
 [2021-10-28 10:16:28] hidapi_try_lib: Loaded libhidapi-hidraw.so, but no devices enumerated; trying other libraries
 [2021-10-28 10:16:28] hidapi_try_lib: Loaded libhidapi-libusb.so, but no devices enumerated; trying other libraries
 [2021-10-28 10:16:28] hid_devinfo_scan: Failed to load any hidapi library
 [2021-10-28 10:16:28] lowlevel_scan: Found usb device at usb:001:001 (path=(null), vid=1d6b, pid=0002, manuf=Linux 5.10.17-v7+ dwc_otg_hcd, prod=DWC OTG Controller, serial=3f980000.usb)
 [2021-10-28 10:16:28] lowlevel_scan: Found usb device at usb:001:002 (path=(null), vid=0424, pid=9514, manuf=(null), prod=(null), serial=(null))
 [2021-10-28 10:16:28] lowlevel_scan: Found usb device at usb:001:003 (path=(null), vid=0424, pid=ec00, manuf=(null), prod=(null), serial=(null))
 [2021-10-28 10:16:28] No devices detected!
 [2021-10-28 10:16:28] Waiting for devices
```

Still no luck

**Comment by luke-jr** [comment]
You might want to check out the README files. Especially the `-S` option
